packages/sales-api-service/src/services/transactions/process-transaction-queue.js

```diff
@@ -142,27 +146,6 @@ const mapToPermission = async (
   return permission
 }
 
-/**
- * Process a recurring payment instruction
- * @param transactionRecord
- * @returns {Promise<{recurringPayment: null, payer: null}>}
- */
-const processRecurringPayment = async transactionRecord => {
-  let recurringPayment = null
-  let payer = null
-  if (transactionRecord.payment.recurring) {
-    const inceptionMoment = moment(transactionRecord.payment.timestamp, true).utc()
-    recurringPayment = new RecurringPayment()
-    recurringPayment.referenceNumber = transactionRecord.payment.recurring.referenceNumber
-    recurringPayment.mandate = transactionRecord.payment.recurring.mandate
-    recurringPayment.inceptionDay = inceptionMoment.date()
-    recurringPayment.inceptionMonth = inceptionMoment.month()
-    payer = await resolveContactPayload(transactionRecord.payment.recurring.payer)
-    recurringPayment.bindToEntity(RecurringPayment.definition.relationships.payer, payer)
-  }
-  return { recurringPayment, payer }
-}
```

Comment:
maybe I'm missing something, but as you've removed the processRecurringPayment function, why are you still calling it on line 72?

Reply:
I moved to recurring-payments service and called it line 72 instead higher up as had access to contact